prow/plugins/trigger/pr.go

```diff
@@ -83,7 +84,7 @@ I'm waiting for a [%s](https://github.com/orgs/%s/people) member to verify that
 %s
 </details>
 `
-	comment := fmt.Sprintf(commentTemplate, pr.User.Login, trustedOrg, trustedOrg, plugins.AboutThisBot)
+	comment := fmt.Sprintf(commentTemplate, pr.User.Login, strings.Join(trustedOrgs, ","), strings.Join(trustedOrgs, ","), plugins.AboutThisBot)
```

Comment:
I changed the template to make more sense.  Let me know if you want different wording.